user-interface/0001-WALL-wallet.md

```diff
@@ -7,7 +7,7 @@ A wallet can contain many public/private key pairs. The public part of each key
 When on the wallet page of Vega.xyz, I...
```

Comment:
GH won't let me comment further up, but my comment is about line 4. 

>  The primary job of a wallet app is to authenticate a user's actions.

I'd say the primary job(s) are to sign and send transactions. I'm not sure if 'authentication' really covers all of that, not if you're just reading the sentence.

(Also the spec link can just be /specs/ now)